src/api/procedures/types.ts

```diff
@@ -435,15 +435,24 @@ export interface AddInstructionParams {
    * date at which the trade was executed (optional, for off chain trades)
    */
   valueDate?: Date;
-  /**
-   * block at which the Instruction will be executed automatically (optional, the Instruction will be executed when all participants have authorized it if not supplied)
-   */
-  endBlock?: BigNumber;
   /**
    * identifier string to help differentiate instructions
    */
   memo?: string;
-}
+} & (
+  | {
+      /**
+       * block at which the Instruction will be executed automatically (optional, the Instruction will be executed when all participants have authorized it if not supplied)
+       */
+      endBlock?: BigNumber;
+    }
+  | {
+      /**
+       * block after which the Instruction will be manually executed (optional, the Instruction will be executed when all participants have authorized it if not supplied)
```

Comment:
```suggestion
       * block after which the Instruction can be manually executed (optional, the Instruction can be executed when all participants have authorized it if not supplied)
```

`will` contradicts the word manual here